package.json

```
@@ -66,7 +66,7 @@
     "docs:dev:run": "vitepress dev docs",
     "docs:serve": "vitepress serve docs",
     "format": "prettier --write .",
-    "lint": "eslint .",
+    "lint": "eslint --cache --cache-strategy content .",
```

Let's do it this way, so users can decide to use it or not, also the CI and lint-stage will not get affected by it and always use a clean lint run

```suggestion
    "lint": "eslint .",
    "lint:cache": "eslint --cache --cache-strategy content .",
```

---

https://github.com/eslint/eslint/issues/10712#issuecomment-409748549

---

First of all the issue is 4y old and we are not using any of those plugins. Saying that, I am using `node` and `import` together with enabled cache and none of them ever caused any issues. I am not sure if anything changed since the linked issue was created and that's why I am not experiencing problems. But again we do not use those plugins and I would rather like to see a proof of malfunction than just make it very slow for everyone linting locally...